config/initializers/airbrake.rb

```diff
@@ -8,6 +8,7 @@
     config.project_key = key
 
     config.blacklist_keys = Rails.application.config.filter_parameters + ['HTTP_AUTHORIZATION']
+    config.ignore << "Sinatra::NotFound"
```

Comment:
double is simpler to change ... single is weird and does not give any perf benefit .. so sticking with double :D